pkg/assetfetcher/asset.go

```diff
@@ -316,6 +316,9 @@ func (a Asset) AssetPayloadToAssetEvent() (domain.AssetEvent, error) {
 	if err != nil {
 		return domain.AssetEvent{}, err
 	}
+	if a.ID == 0 || a.IP == "" || lastScanned.IsZero() {
```

Comment:
task: a test with an asset that is missing a required field that results in an error

Reply:
How's that?